### PR TITLE
Adding node version v10.15.3 for u16node images 

### PR DESCRIPTION
**Pull request by Ranjansingh41**
https://github.com/Shippable/pm/issues/12307
image

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+echo "================= Installing basic packages ==================="
+apt-get update
+apt-get install -y \
+curl \
```
Comment (rageshkrishna) [inline install.sh]: Why `\` at the end here? If it's only one package, why not just keep it on line 5 for now?

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+export NVM_VERSION=v0.33.9
+echo "================= Installing NVM "$NVM_VERSION" ==================="
+curl -sS https://raw.githubusercontent.com/creationix/nvm/"$NVM_VERSION"/install.sh | bash
+
+# Set NVM_DIR so the installations go to the right place
+export NVM_DIR="/root/.nvm"
+
+# add source of nvm to .bashrc - allows to use nvm as a command
+echo "source ~/.nvm/nvm.sh"
```
Comment (rageshkrishna) [inline install.sh]: This isn't doing anything.

Comment (rageshkrishna) [reply install.sh]: Doesn't this need to go to a file which is then referenced in `$BASH_ENV`?

**Review by rageshkrishna — CHANGES_REQUESTED** [review]

File: install.sh (hunk 1 above)
Comment (Ranjansingh41) [reply install.sh]: ok i will update

File: install.sh (hunk 2 above)
Comment (Bharath92) [reply install.sh]: is it required? we already seem to be running running ~/.nvm/nvm.sh here https://github.com/Shippable/kermit-execTemplates/blob/master/steps/bash/header.sh#L1396

Comment (rageshkrishna) [reply install.sh]: Makes sense. We should remove this.

Comment (Ranjansingh41) [reply install.sh]: ok i will remove this

**Review by rageshkrishna — APPROVED** [review]

**Pull request merged**